packages/fluentui/react-northstar/src/components/Menu/Menu.tsx

```diff
@@ -199,13 +213,20 @@ export const Menu = compose<'ul', MenuProps, MenuStylesProps, {}, {}>(
       setIndex(activeIndex);
     };
 
-    const handleItemOverrides = predefinedProps => ({
-      onClick: (e, itemProps) => {
+    const handleClick = React.useCallback(
+      (e, itemProps) => {
         const { index } = itemProps;
 
         setActiveIndex(e, index);
 
-        _.invoke(props, 'onItemClick', e, itemProps);
+        _.invoke(actualProps.current, 'onItemClick', e, itemProps);
+      },
+      [actualProps, setActiveIndex],
```

Comment:
Actually here we don't need all props, we need only `onItemClick` from the props. You can replace the `_.invoke` with 
```
onItemClick && onItemClick(e, itemProps);
```

Reply:
Done

Reply:
If `onItemClick` will be close it will be unstable: `<Menu onItemClick={() => {}} />